app/src/main/java/com/example/neighborly/JoinBuildingActivity.java

```diff
@@ -35,7 +35,7 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_join_building);
         myDialog = new Dialog(this);
 
-        buildingImage = (CircleImageView) findViewById(R.id.buildingImage);
+        buildingImage = (CircleImageView) findViewById(R.id.addItemImage);
         buildingImage.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
```

Comment:
empty?

Reply:
only for **debugging** as we currently don't support a db